### PR TITLE
Lunar Starship Tutorial update

### PR DESCRIPTION
**Pull request by gtep96**
Requesting review

### CODE REVIEW

File: docs_tutorials/lunar_starship/lunar_starship.md [hunk 1]
```diff
@@ -4,311 +4,181 @@ sidebar_label: 'Lunar Starship'
...
+## Scenario
+For this tutorial, let's check how far the Starship can fly depending on Specific Impulse.<br/>
+This example provides an array of time dependent values of states computed by solving non-linear optimal control problems(OCP) in the standard Bolza form using pseudo-spectral collocation methods and adjusted using an additional real dynamic function. The OCP solver used in this example is [MPOPT](https://mpopt.readthedocs.io/en/latest/) (based on [IPOPT](https://en.wikipedia.org/wiki/IPOPT#:~:text=IPOPT%2C%20short%20for%20%22Interior%20Point,the%20EPL%20(formerly%20CPL).)) library modified by Lulav Space team. You can define a "real" dynamic function to test the control values computed by solving non-linear optimal control problems(OCP). This function should have the same number of outputs.<br/>
```
[the remark this answers is not in the document]
Comment (gtep96) [reply docs_tutorials/lunar_starship/lunar_starship.md:83]: Done

File: docs_tutorials/lunar_starship/lunar_starship.md [hunk 2]
```diff
@@ -4,311 +4,181 @@ sidebar_label: 'Lunar Starship'
...
+## CITROS Initialization
+1. [Install CITROS](https://citros.io/doc/docs_tutorials/getting_started/#installation).
+2. Follow [these steps](https://citros.io/doc/docs_tutorials/getting_started/#initialization) to Initialize CITROS.
 
-2. Configuring the setup. We need to set up the maximum performance available: timeout, CPU, GPU and Memory. To perform it, we need to define them in ```.citros/simulations/simulation_lunar_starship.json```. The recommended setup is minimum 600 seconds timeout, 4 CPU, 4 GPU and 4096 MB of Memory. Don't forget to save the file!
+Now you can see .citros directory in the explorer.
 
-3. Configuring the params setup. You can find the default setup in ```.citros/parameter_setups/default_param_setup.json```. [CITROS CLI](https://citros.io/doc/docs_cli) provides an opportunity to use basic NumPy functions (such as distributions) and even user-defined functions, but let's keep it default for now. Variables with '_0' are initial conditions, and variables with '_f' are final conditions respectively. Don't forget to save the file!
+## Scenario
+For this tutorial, let's check how far the Starship can fly depending on Specific Impulse.<br/>
```
[the remark this answers is not in the document]
Comment (gtep96) [reply docs_tutorials/lunar_starship/lunar_starship.md:82]: Done

File: docs_tutorials/lunar_starship/lunar_starship.md [hunk 3]
```diff
@@ -4,311 +4,181 @@ sidebar_label: 'Lunar Starship'
...
+This example provides an array of time dependent values of states computed by solving non-linear optimal control problems(OCP) in the standard Bolza form using pseudo-spectral collocation methods and adjusted using an additional real dynamic function. The OCP solver used in this example is [MPOPT](https://mpopt.readthedocs.io/en/latest/) (based on [IPOPT](https://en.wikipedia.org/wiki/IPOPT#:~:text=IPOPT%2C%20short%20for%20%22Interior%20Point,the%20EPL%20(formerly%20CPL).)) library modified by Lulav Space team. You can define a "real" dynamic function to test the control values computed by solving non-linear optimal control problems(OCP). This function should have the same number of outputs.<br/>
+The output of the simulation comprises critical flight data, such as altitude, velocity, and other relevant parameters, recorded over time intervals. These results are published via ROS 2 topics, allowing for real-time data visualization, analysis, and integration with other ROS-based systems.<br/>
+The parameter setup is listed in ```.citros/parameter_setups/default_param_setup.json```.<br/>
```
[the remark this answers is not in the document]
Comment (gtep96) [reply docs_tutorials/lunar_starship/lunar_starship.md:86]: Done

File: docs_tutorials/lunar_starship/lunar_starship.md [hunk 4]
```diff
@@ -4,311 +4,181 @@ sidebar_label: 'Lunar Starship'
...
+ To find out how the traveled distance will change, we need to launch a batch with several simulations and a distribution for Specific impulse parameter, starting from 200 and up to 300.
```
[the remark this answers is not in the document]
Comment (gtep96) [reply docs_tutorials/lunar_starship/lunar_starship.md:87]: Done

**Comment by gtep96** [comment]
@mariaBer updated and pushed

File: docs_tutorials/lunar_starship/lunar_starship.md [hunk 5]
```diff
@@ -4,311 +4,181 @@ sidebar_label: 'Lunar Starship'
...
 
-    |Launch File	|Package	|Description
-    |--|--|--
-    lunar_starship.launch.py	|lunar_starship	|Lunar Starship simulation launch file 	
+Learn more about parameter setup and defining custom functions in [Directory parameter_setups](https://citros.io/doc/docs_cli/structure/citros_structure/#directory-parameter_setups) and [Adding Functions to Parameter Setup](https://citros.io/doc/docs_cli/configuration/config_params) pages.
 
-:::tip
+In addition to parameter setup, you can configure the simulation performance setup (timeout, CPU, GPU and Memory) as well.
+This parameters can be found in ```.citros/simulations/simulation_lunar_starship.json```. <br/>
+The default setup is 600 seconds timeout, 4 CPU, 4 GPU and 4096 MB of Memory.
```
Comment (mariaBer) [inline docs_tutorials/lunar_starship/lunar_starship.md:137]: image

File: docs_tutorials/lunar_starship/lunar_starship.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (mariaBer) [reply docs_tutorials/lunar_starship/lunar_starship.md]: You can define a "theoretical" and a "real" dynamic function to test the control values computed by solving non-linear ...

Is that what we are doing in this tutorial? where did the 7 variables came from?

File: docs_tutorials/lunar_starship/lunar_starship.md [hunk 6]
```diff
@@ -4,311 +4,181 @@ sidebar_label: 'Lunar Starship'
...
+To plot the local run results you can use [Foxglove](https://citros.io/doc/docs_tutorials/#visualization-with-foxglove).
 
-CITROS Web provides a powerful data analysis package, which is a comprehensive solution for data query, analysis and visualization. With its extensive features, you can quickly and easily extract valuable insights from your data. To use it, Jupiter Notebook support is built-in. 
-Navigate to our ```Code``` project page, open the Notebooks folder and click on the notebook file. Here you can see the usual Jupiter editor interface: you can add blocks of code or built-in Markdown engine, run and save notebook and control the Python kernel.
-
-You can find all the data analysis package guides and API reference [here](https://citros.io/doc/docs_data_analysis).
-
-Let's quickly go through the key points of using a Jupiter Notebook and fetching data from a database. But to try some brief examples of data analysis using the built-in package, we need to launch a batch with several simulations and a distribution for one of the ROS parameters (Specific impulse, in our case). This parameter will be different for each simulation:
+![png](img/img0.png "FoxGlove example")
```
Comment (mariaBer) [inline docs_tutorials/lunar_starship/lunar_starship.md:162]: These are the names I see, unlike in the image in the tutorial:
image

**Review by mariaBer — CHANGES_REQUESTED** [review]

File: docs_tutorials/lunar_starship/lunar_starship.md (hunk 5 above)
Comment (gtep96) [reply docs_tutorials/lunar_starship/lunar_starship.md:137]: @mariaBer https://github.com/lulav/citros_doc/pull/101#discussion_r1431885192

File: docs_tutorials/lunar_starship/lunar_starship.md (hunk 6 above)
Comment (gtep96) [reply docs_tutorials/lunar_starship/lunar_starship.md:162]: Cant see what did you send
image

**Comment by gtep96** [comment]
updated and pushed

File: docs_tutorials/lunar_starship/lunar_starship.md (hunk 6 above)
Comment (mariaBer) [reply docs_tutorials/lunar_starship/lunar_starship.md:175]: Did you check why I dont receive the specific name of the filed but just the lunar_startship/state.data[n]?

Comment (gtep96) [reply docs_tutorials/lunar_starship/lunar_starship.md:175]: Yes, please pull the changes and check it

**Comment by gtep96** [comment]
Foxglove image updated

**Pull request merged**